### ISSUE TITLE
Support Templater when creating new notes

### ISSUE THREAD

**Issue by taurr** [nhaouari/obsidian-textgenerator-plugin#209]
it would be really nice if we had support for Templater templates when generating into a new Note.

**Comment by TrueMilli** [comment]
I'd also like this!
It already provides syntax for this, we "only" need a `command`. 
```
app.commands.executeCommandById(command)
```

**Comment by taurr** [comment]
Syntax for it — how?

**Comment by TrueMilli** [comment]
I found it. This calls textgenerator from Templater:
```
<%* await app.commands.executeCommandById("obsidian-textgenerator-plugin:generate-text") %>
```

But I'm not sure how to configure what gets passed. Since a call to the OpenAI API is not too complicated, I just wrote my own script, feel free to take it as basis for whatever you want to do: https://github.com/TrueMilli/Obsidian-Templates/blob/main/Templater%20Scripts/describe.js

Here's how you call it.
```
<% await tp.user.describe(title, creator, year) %>
```

In case you want to fiddle around with text generator:
This dataview gives you all commands available to you (considering YOUR plugins): https://forum.obsidian.md/t/dataviewjs-snippet-showcase/17847/37

Let me know how it goes for you!
